geonode/utils.py

```diff
@@ -1069,6 +1069,13 @@ def printsignals():
 def designals():
     global signals_store
 
+    if not signals_store:
```

Comment:
I don't understand where they initialize this. I don't see it anywhere in the code.
So, we initialize this in this step before designaling all signals.